kured-ds-unprivileged.yaml

```diff
@@ -0,0 +1,140 @@
+---
```

Comment:
Is there any drawback so that we do not make this the new default?

Reply:
I don't see any major problem on pushing this to be the default. 
The new approach won't break existing workloads and we could update the `yaml` and helm chart to be a seamless replacement. 

The only down side of the new approach is having two `daemonset`s, which in my opinion is a tiny downside when considering we will finally be able to run kured unprivileged and with a single Linux capability.

Shall we get this merged and then a follow up PR to just make it the default so we can hear more feedback just around that possibility?

Reply:
Yeah sounds reasonable, I tend to make it the default. 👍